t/router/radixtree-sni2.t

```diff
@@ -354,3 +354,47 @@ failed to do SSL handshake: handshake failed
 failed to fetch ssl config: failed to find SNI: please check if the client requests via IP or uses an outdated protocol
 --- no_error_log
 [alert]
+
+
+
+=== TEST 9: client request without sni, but fallback_sni is set
+--- yaml_config
+apisix:
+  node_listen: 1984
+  ssl:
+    fallback_sni: "a.test2.com"
+--- config
+listen unix:$TEST_NGINX_HTML_DIR/nginx.sock ssl;
+
+location /t {
```

Comment:
What is the `location /t { /t {` style, it looks so strange, all the following ones are this repeated nested layer.

Reply:
? There is no `location /t { /t {`.

Reply:
ignore this, it was there the last time I saw it, but it's not there now.